charts/asset-manager/templates/freshclam-cronjob.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ .Release.Name }}-freshclam
+spec:
+  schedule: "19 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          {{- include "asset-manager.freshclam.podspec" . | indent 8 }}
+          restartPolicy: OnFailure
```

optional: `Never` might be preferable here since it'll make it easier to debug failures and we'll try again automatically the next hour anyway.
```suggestion
          restartPolicy: Never
```

---

Ok, I will change that.